cmake/O2Dependencies.cmake

```diff
@@ -1310,7 +1310,7 @@ o2_define_bucket(
     DEPENDENCIES
     #-- buckets follow
     fairroot_base_bucket
-    pythia8
+    # pythia8
```

Comment:
@sawenzel, if it's not needed it can be removed completely, or do you think it's just temporarily disabled?